outlines/ui-ux-outline.md

```diff
@@ -0,0 +1,32 @@
+# UI / UX
+
+1. Building reusable components to encapsulate UX patterns
+  1. Pure vs impure in general
```

Comment:
Yeah I like "smart"/"pure" - then everyone can feel good, each type of component has its own merits!